specs-go/config.go

```diff
@@ -44,8 +44,8 @@ type Process struct {
 	// Cwd is the current working directory for the process and must be
 	// relative to the container's root.
 	Cwd string `json:"cwd"`
-	// Capabilities are Linux capabilities that are kept for the container.
-	Capabilities []string `json:"capabilities,omitempty" platform:"linux"`
+	// Capabilities are Linux capabilities that are kept for the process.
+	Capabilities *LinuxCapabilities `json:"capabilities,omitempty" platform:"linux"`
```

Comment:
> Solaris capabilities are a bit similar by the look of it (they have E, I, P but different capabilities) but that doesn't mean they should be at the top level.

Is that “I don't think there is enough overlap to be worth sharing the same namespace”?  If the Solaris folks are ok using `bounding` instead of their `limited`, we could just make `ambient` Linux-specific.  In other situations where only the values are different (e.g. [`mounts` entries][1]), the maintainer preference seems to be “share the same property, but define the different per-platform values”.

On the other hand, if Windows has nothing equivalent, the [pro-namespacing policy here][2] suggests namespacing the whole `capabilities` structure to something POSIX-ish-specific.  I tried to raise the “[how will this be handled on Windows?][3]” issue in #673, but am still not clear on if/whether Windows supports something capability-like.

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc4/config.md#mounts
[2]: http://ircbot.wl.linuxfoundation.org/meetings/opencontainers/2016/opencontainers.2016-05-18-17.01.log.html#l-101
[3]: https://github.com/opencontainers/runtime-spec/pull/673#discussion_r99353136